src/plugins/mf_priority.cpp

```diff
@@ -60,6 +62,16 @@ std::map<std::string, int> priority_weights;
  *
  * queue: a factor that can further affect the priority of a job based on the
  *     queue passed in.
+ *
+ * age: a factor that considers the time that a job was released by the
+ *     scheduler to be run but could not due to another constraint (e.g a
+ *     resource constraint). Any urgency, priority, or jobspec-update event
```

Comment:
This statement is unclear to me, but I may just be reading it wrong. What do you mean by "released by the scheduler to run"?

Would it be clearer to just say: "The age factor considers the most recent timestamp that a job entered the SCHED state and therefore represents the amount of time the job has been eligible for scheduling." Or is that statement incorrect (I haven't fully reviewed this PR yet, so apologies if so.)

Reply:
Oh, maybe you just mean "released _to_ the scheduler"?

Reply:
Oops, yes I mean _to_ the scheduler here, not _by_. I can fix this - thanks!